tutor/src/screens/new-course/ux.js

```diff
@@ -91,9 +91,9 @@ export default class CourseBuilderUX extends BaseModel {
   }
 
   @action.bound onOfferingsAvailable() {
-      if (this.preselectedAppearanceCode) {
+      if (this.preselectedOfferingId) {
           this.newCourse.offering = find(this.offerings.available.array,
-              { appearance_code: this.preselectedAppearanceCode }
+              { offering_id: this.preselectedOfferingId }
           );
```

Comment:
@camliu89 sorry I should have tested locally before approving but this has a bug :(   offerings do not have `offering_id`, they have just `id`, so this should be: 
```
this.newCourse.offering = find(this.offerings.available.array,
               { id: this.preselectedOfferingId }
           );
```
However! we've ran into problems with using lodash find before, it uses triple equal and will fail if id is a number and `preselectedOfferingId` is a string.

How about:
```
this.newCourse.offering = this.offerings.available.array.find(o => o.id == this.preselectedOfferingId)
```

Reply:
yeah, that is correct. thanks nathan.